dotfiles/install_crictl.sh

```diff
@@ -0,0 +1,25 @@
+if ! [ $(id -u) = 0 ]; then
+    echo "running as non-root and crictl will not be installed"
+    exit 0
+fi
+
+if [ -x "$(command -v crictl)" ]; then
+rm -rf /etc/crictl.yaml
+tee /etc/crictl.yaml << EOT
+  runtime-endpoint: unix:///run/containerd/containerd.sock
+  image-endpoint: unix:///run/containerd/containerd.sock
+  timeout: 10
+EOT
+exit 0
+fi
+
+VERSION="v1.17.0"
+curl -fsSLO https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz
+tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
+rm -f crictl-$VERSION-linux-amd64.tar.gz
+rm -rf /etc/crictl.yaml
+tee /etc/crictl.yaml << EOT
+runtime-endpoint: unix:///run/containerd/containerd.sock
+image-endpoint: unix:///run/containerd/containerd.sock
```

Comment:
I just played around with `crictl` - even without running `chroot` we can configure this file with the following options:
```
runtime-endpoint: unix:///host/run/dockershim.sock
image-endpoint: unix:///host/run/dockershim.sock
```
Since the node's root directory is mounted under /host. With this config `crictl` is directly connected to the docker daemon running on the node (in future we can change this to another runtime) and we can debug containers from the ops-pod without having to execute `chroot` and instal additional binaries or overwrite files directly on the node.

Reply:
`gardenctl` and the `terminal-controller-manager` mount the node's root dir under `/hostroot`

Reply:
we can change the `ops-pod` script to also mount it under /hostroot, also add this to the /etc/crictil.yaml and mention it in the help info for crictl